tests/data/test_AtomicData.py

```diff
@@ -19,6 +19,33 @@ def test_from_ase(CuFcc):
         assert data[key].shape == (len(atoms), 3)  # 4 species in this atoms
 
 
+def test_to_ase(CH3CHO):
+    atoms, data = CH3CHO
+    to_ase = data.to_ase()
+    to_ase_atoms = to_ase[0]
+    assert np.allclose(atoms.get_positions(), to_ase_atoms.get_positions())
+    assert np.array_equal(atoms.get_atomic_numbers(), to_ase_atoms.get_atomic_numbers())
+    assert np.array_equal(atoms.get_pbc(), to_ase_atoms.get_pbc())
+    assert np.array_equal(atoms.get_cell(), to_ase_atoms.get_cell())
+
+
+def test_to_ase_batches():
+    atomic_numbers = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_atomic_numbers.npz")['arr_0']
+    pos = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_positions.npz")['arr_0']
+    batch = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_batch.npz")['arr_0']
+    cell = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_cell.npz")['arr_0']
+    pbc = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_pbc.npz")['arr_0']
+    edge_index = np.load("C:/Users/alber/nequip/nequip/scripts/aspirin_edge_index.npz")['arr_0']
+    data = AtomicData(pos=pos, atomic_numbers=atomic_numbers, batch=batch, cell=cell, pbc=pbc, edge_index=edge_index)
```

Comment:
as discussed, let's replace this with the `atomic_batch` fixture

(which, if curious, is defined L78 of conftest.py)